pkg/workflow/controller/config.go

```diff
@@ -98,6 +100,17 @@ type LimitsConfig struct {
 	MaxWorkflowRuns int `json:"max_workflowruns"`
 }
 
+// NotificationEndpoint represents the config of notification endpoint.
+// Workflow controller will send notifications after workflowruns finish
+// if notification endpoints are configured.
+type NotificationEndpoint struct {
+	// Name represents the name of notification endpoint.
+	Name string `json:"name"`
```

Comment:
Where do we need this `Name` ?

Reply:
We support multiple notification endpoints, need a name to easily distinguish them.

Reply:
We can't distinguish them by URL ? With name, users have to guarantee uniqueness of names.

Reply:
We do not depend on the name, is for users. We can make this as optional.

Reply:
So name is just a display name for users to read ?